src/providers/setting-service.ts

```diff
@@ -73,7 +73,7 @@ export class SettingService {
   }
 
   isShowRestUrlField(): Observable<boolean> {
-    return this.readKey(this.isShowRestUrlFieldKey).map(val => val == null ? this.isShowRestUrlFieldDefault : val);
+    return this.readKey(this.isShowRestUrlFieldKey).map(val => val === undefined ? this.isShowRestUrlFieldDefault : val);
```

Comment:
I think undefined is fine here but the method readKey should map null to undefined and then return a real respone object instead of any.